site/docs/recipes/service-discovery.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Service discovery using environment variables"
+sidebar_position: 1
+date: "2023-05-07"
+---
+
+## Problem
+
+In development environments, it's common for frontend applications to communicate with backend services using exposed ports. While Preevy handles service-service communication well with compose service discovery, it can be challenging when frontend applications need to connect to backend services via exposed ports. The tunneling URL must be replaced, but it's not known in advance.
```

Comment:
I don't think "compose service discovery" is a familiar term.

Maybe add explicit explanation that code running in the containers can access other containers using the name of the service as a hostname (https://docs.docker.com/compose/networking/), but code running in the browser does not.